.github/workflows/scripts/win-test-services.ps1

```diff
@@ -1,29 +1,46 @@
 param (
     [string]$mode = "agent",
+    [string]$access_token = "testing123",
+    [string]$realm = "test",
+    [string]$memory = "512",
     [string]$with_fluentd = "true"
 )
 
 $ErrorActionPreference = 'Stop'
-Set-PSDebug -Trace 1
+# Set-PSDebug -Trace 1
```

Comment:
Similar suggestion to the CI workflow: use a parameter with the desired default.